finetune/utils/data/raw_datasets.py

```diff
@@ -154,6 +154,11 @@ def get_eval_data(self):
         if self.raw_datasets["eval"] is not None:
             return self.raw_datasets["eval"]
         return None
+    
+    def get_prompt(self, sample):
+        if sample["prompt"] is not None:
+            return " " + sample["prompt"]
```

Comment:
why do we add an extra space here?

Reply:
this is a routine inherited from data_utils from ds-chat

Reply:
got it